admin/adminChatAnalyser.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+$countryID1 = $countryID2 = array();
+$gameID=$userID=0;
+$reason = 'Press was checked for gameID: ';
+
+if (!defined('IN_CODE'))
+{
+	chdir ('..');
+	require_once('header.php');
+}
+
+if (!$User->type['Moderator'])
```

Comment:
I think this should be the first line in the file

Reply:
Doing so breaks the exports, but I have confirmed that you cannot access any part of this page when not logged in or when logged in as not a mod.

Reply:
It is currently the case that you can't use this page while not a moderator, but to avoid bugs in the future, this guard is better placed at the top of the file.

I'm not sure what you mean by breaks the exports?